### PR TITLE
Twilight 2000 4e (T2K-4e) Character Sheet: twilight-2k-4e.html

### PR DESCRIPTION
**Pull request by NocFenix**
## Changes / Comments

Added Twilight 2000 4th Edition Character Sheet.
This sheet is based on the current beta rules-set available.

The sheet includes images for dice template rolling, as well as the game's logo. It also includes tracking of basic stats, inventory, weapons, and vehicles. Overall, this gives players all of the basic features they will need to get started with T2K 4e on Roll20.

## Roll20 Requests

Comments are very helpful for reviewing the code changes. Please answer the relevant questions below in your comment.

- [x] Does the pull request title have the sheet name(s)? Include each sheet name.
- [x] Is this a bug fix?
- Not applicable as it is a new sheet
- [x] Does this add functional enhancements (new features or extending existing features) ?
- [x] Does this add or change functional aesthetics (such as layout or color scheme) ? 
- [x] If changing or removing attributes, what steps have you taken, if any, to preserve player data ?
- Not applicable as it is a new sheet
- [x] If this is a new sheet, did you follow [Building Character Sheets standards](https://wiki.roll20.net/Building_Character_Sheets#Roll20_Character_Sheets_Repository) ?

If you do not know English. Please leave a comment in your native language.

### CODE REVIEW

**Comment by Lockbox313** [comment]
What is the issue with this sheet?

**Comment by NocFenix** [comment]
@Lockbox313, I wish I knew. When I tried to add it to the approved.yaml myself it growled at me saying I shouldn't change the root directory, so I reverted that. Made some other fine tunings and figured I'd met all the requirements, but it still growls at me. Not sure who I should be pinging to get it approved.

**Comment by Kurohyou** [comment]
@wing-it, This looks like it just needs some dev time to process the manual part of the update maybe?

**Comment by Kurohyou** [comment]
Also, @NocFenix, are you sure you want this to **not** use the legacy CSS sanitization? The new sanitization has not gone live yet and your sheet won't work (and likely won't be accepted) until the new sanitization does go live.

**Comment by NocFenix** [comment]
@Kurohyou, how would it not work? For my testing I used both the Custom Sheet Sandbox as well as a game with just Custom Sheet. Unless I am missing something important.

**Comment by Lockbox313** [comment]
You can't added it to the approved.yaml only Roll20 people can do that, that probably what the problem is,

**Comment by Lockbox313** [comment]
> @Kurohyou, how would it not work? For my testing I used both the Custom Sheet Sandbox as well as a game with just Custom Sheet. Unless I am missing something important.



> @Kurohyou, how would it not work? For my testing I used both the Custom Sheet Sandbox as well as a game with just Custom Sheet. Unless I am missing something important.

Did you set the legacy": false in the sheet.json? It should be set to true.

File: T2K 4e/sheet.json [hunk 1]
```diff
@@ -0,0 +1,9 @@
+{
+    "html": "twilight-2k-4e.html",
+    "css": "twilight-2k-4e.css",
+    "authors": "/u/NocFenix(3266353)",
+    "roll20userid": "3266353",
+    "preview": "twilight-2k-4e.png",
+    "instructions": "You can click on the Attribute names to roll against your attributes.",
+    "legacy": false
```
Comment (Anduh) [inline T2K 4e/sheet.json:8]:
```suggestion
    "legacy": true
```
This is the fix that people are proposing in this thread, as Roll20 haven't yet implemented the new sheet stuff.

You should be able to just approve my edit as is

**Comment by NocFenix** [comment]
Alright. I guess I'll just wait. I changed legacy to true, I've removed all of the sheet- from the classes in the html (except the roll template). Not sure what else to check to get this more ready.

**Pull request merged**